package.json

```diff
@@ -20,11 +20,9 @@
     "Other"
   ],
   "activationEvents": [
+    "onStartupFinished",
     "onLanguage:python",
-    "onCommand:shiny.python.runApp",
-    "onCommand:shiny.python.debugApp",
-    "onLanguage:r",
-    "onCommand:shiny.r.runApp"
+    "onLanguage:r"
```

Comment:
Because we are now activating `onStartupFinished` and `onLanguage:python` or `onLanguage:r`, I don't think we need the `onCommand` activation events